src/core/services/modals.ts

```diff
@@ -85,6 +92,11 @@ export class CoreModalsService {
 
         viewContainer?.setAttribute('aria-hidden', 'true');
 
+        if (backdropDismiss) {
+            const backdrop = document.getElementsByTagName('ion-backdrop')?.[0];
```

Comment:
```suggestion
            const backdrop = element.querySelector('ion-backdrop');
```
It's better to search inside `element` to find the proper backdrop, since the app could be displaying more than one.

Also, if you use querySelector you can get rid of the "?.[0]" part.